packages/diagrams-demo-gallery/demos/demo-zoom-to-fit-nodes/index.tsx

```diff
@@ -0,0 +1,55 @@
+import createEngine, { DiagramModel, DefaultNodeModel } from '@projectstorm/react-diagrams';
+import * as React from 'react';
+import { DemoWorkspaceWidget, DemoButton } from '../helpers/DemoWorkspaceWidget';
+import { CanvasWidget } from '@projectstorm/react-canvas-core';
+import { DemoCanvasWidget } from '../helpers/DemoCanvasWidget';
+
+/**
+ *
+ * Simple stress test of the system plus zoom to fit function
+ *
+ * @Author Dylan Vorster
```

Comment:
im not the author :P

Reply:
regarding the changes I've made on this file, it's almost like it ;)